### PR TITLE
Enable multithreaded video decoding

### PR DESCRIPTION
**Pull request by lukasf**
This should solve #192.

### CODE REVIEW

**Comment by mcosmin222** [comment]
I will try to test this out tomorrow evening.

File: FFmpegInterop/Source/FFmpegInteropMSS.cpp [hunk 1]
```diff
@@ -399,6 +409,14 @@ HRESULT FFmpegInteropMSS::InitFFmpegContext(bool forceAudioDecode, bool forceVid
 
 				if (SUCCEEDED(hr))
 				{
+					// enable multi threading
+					unsigned threads = std::thread::hardware_concurrency();
+					if (threads > 0)
+					{
+						avVideoCodecCtx->thread_count = threads;
+						avVideoCodecCtx->thread_type = FF_THREAD_FRAME || FF_THREAD_SLICE;
```
Comment (khouzam) [inline FFmpegInterop/Source/FFmpegInteropMSS.cpp]: This should be a bitwise or instead of a logical or. `FF_THREAD_FRAME | FF_THREAD_SLICE`

Comment (lukasf) [reply FFmpegInterop/Source/FFmpegInteropMSS.cpp]: Absolutely. Fixed now.

**Comment by mcosmin222** [comment]
OK. No obvious breaking changes.

**Comment by khouzam** [comment]
Thanks for code and validation.

**Pull request merged**